### ISSUE TITLE
Re-Add Enderdragon pets

### ISSUE THREAD

**Issue by andrewkm** [TechzoneMC/SonarPet#23]
http://i.imgur.com/uUDNoqQ.png

Spigot 801
Sonarpet 17

Let me know if you need more information. Simply ain't working.

**Comment by andrewkm** [comment]
Same goes for human pet.

**Comment by Techcable** [comment]
This was intentionally removed, because I'd have to re-implement ender dragon AI (see issue #25). If we fix #25, then we can utilize the existing NMS logic.
Human pet is a duplicate of #26

**Comment by andrewkm** [comment]
Closing issue.
Referenced here: https://github.com/TechzoneMC/SonarPet/issues/26

**Comment by Techcable** [comment]
What, this is a valid issue?

**Comment by andrewkm** [comment]
Oh ok, thought since you referenced to the other one, I'd close this one so as not to spam you issues.

**Comment by Techcable** [comment]
Blocked by #54

**Comment by andrewkm** [comment]
Fixed for me now.
